aiutils/vis/image_io.py

```diff
@@ -1,54 +1,55 @@
-from termcolor import colored
 from PIL import Image
 import numpy as np
 import pdb
 
-
-def raise_exception(e, err_str):
-    pkg_err_str = 'Exception raised in package pyAIUtil: \n{}'.format(err_str)
-    print colored(pkg_err_str, 'red')
-    raise
-
-
-def check_assertion(cond, err_str):
-    pkg_err_str = '\nAssertion failed in package pyAIUtil: \n{}'.format(
-        err_str)
-    assert (cond), colored(pkg_err_str, 'blue')
+import aiutils.devtools.err_msg as err_msg
 
 
 def imread(filename):
+    """
+    Matlab like function for reading an image file.
+
+    Returns:
+    np_im (numpy.ndarray): h x w x 3 ndarray for color images and 
+        h x w for grayscale images
+    """
     try:
         im = Image.open(filename)
-
     except IOError as e:
```

Comment:
Do we need to do this sort of exception handling? I think there are different schools of thought on this. Do you have a strong preference? I prefer letting PIL raise its own exception, and let the end-user use the stack trace to debug. It keeps out code cleaner.